packages/react-ui-ag/src/Listings/__tests__/SingleFamilyMobileMapListing-test.js

```diff
@@ -60,6 +60,20 @@ describe('ag/Listing/SingleFamilyMobileMapListing', () => {
     expect(ctaClick).toHaveBeenCalled()
   })
 
+  it('fires cta buttons on click action when prioritizeCardClick is true', () => {
```

Comment:
Do you think we could add some tests to make sure it doesn't fire the click actions when prioritizeCardClick is false? Since that is the issue we were having before

Reply:
@bethstevic Just talked to @paudyalhari . Seems we do seem to have those tests. Just they are in ListingCell, not here.